definitions/divorce/json/CaseTypeTab/CaseTypeTab-pet-sol-selects-own-org-nonprod.json

```diff
@@ -1,12 +0,0 @@
-[
-    {
-        "LiveFrom": "01/01/2017",
```

Comment:
moved to https://github.com/hmcts/div-ccd-definitions/pull/707/files#diff-a8f22c0373eac3c372081a2abf2465648c58a64d7408196ff8b1949aec1ad12eR183-R192

RRJ release.